src/AdjustSync.cpp

```diff
@@ -79,6 +79,20 @@ void AdjustSync::ResetOriginalSyncData()
 	ResetAutosync();
 }
 
+float AdjustSync::calc_mean_kahan(float* start, float* end)
```

Comment:
Let's add a comment above linking to the wikipedia page: https://en.wikipedia.org/wiki/Kahan_summation_algorithm

And actually, let's just entirely replace the body of calc_mean in RageUtil with this version.

Reply:
Sounds good I like that!